app/helpers/session_helper.rb

```diff
@@ -0,0 +1,5 @@
+module SessionHelper
```

Comment:
Helpers are available on every page, which means having generic method names will become a problem as the application starts growing. I suggest picking a name that relates to the domain of the helper module